lib/ui/lobby.js

```diff
@@ -89,6 +93,11 @@ export default class Lobby {
       ev.preventDefault();
       dispatch(this, 'CreateNewGame', { mode: this.gameModeSelect.value });
     };
+
+    this.nickInput.onchange = this.nickInput.oninput = (ev) => {
```

Comment:
Might be good idea to use [debounce](https://lodash.com/docs/4.17.4#debounce) here to avoid spamming of events.